docs/install-and-setup.md

```diff
@@ -12,12 +12,13 @@ The Angular CLI's `new` command will set up the latest Angular build in a new pr
 
 ### 2. Install AngularFire and Firebase
 
+Now that you have a new project setup, install AngularFire and Firebase from npm.
+
 ```bash
-ng add @angular/fire@next
+ng add @angular/fire
+npm install firebase
```

Comment:
better to save this in application
`npm install firebase --save`

Reply:
`--save` is default since npm 5.0